### PR TITLE
intro page copy edits

### PR DESCRIPTION
**Pull request by rianfowler**
2018-02-14-08-48-localhost_3001

### CODE REVIEW

File: src/js/vic-v2/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -114,25 +113,26 @@ ID.me. To go through the ID.me identity-proofing process, you’ll need:</p>
...
+              <p>We’ll review your application to check your eligibility. We may contact you if we have any questions or need more information.</p>
             </li>
             <li className="process-step list-five">
               <div><h5>Decision</h5></div>
-              <p>Once we verify your application, you should receive your Veteran ID Card by mail in about 60 days.</p>
+              <p>Once we approve your application, you should receive your Veteran ID Card by mail in about 60 days.</p>
             </li>
           </ol>
         </div>
+        <h2 className="help-heading">Ready to get started?</h2>
```
Comment (rianfowler) [inline src/js/vic-v2/components/IntroductionPage.jsx]: Goal was to have this look like the Need help? heading- want to make sure it's okay to reuse the styling and h2 or if I should duplicate it into the Intro page css.

File: src/js/vic-v2/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ class IntroductionPage extends React.Component {
           resumeOnly
           {...this.props.saveInProgressActions}
           {...this.props.saveInProgress}>
-          Please complete the VIC form to apply for a card
+          Please complete the VIC form to apply for a card.
```
Comment (jbalboni) [inline src/js/vic-v2/components/IntroductionPage.jsx]: Do we need to spell out VIC here @peggygannon?

**Review by jbalboni — APPROVED** [review]
LGTM, should have Peggy review the changes, too.

**Comment by rianfowler** [comment]
Ah I merged this before @peggygannon had a chance to review. I'll submit another PR if there are additional changes needed.

**Comment by peggygannon** [comment]
@jbalboni @rianfowler see below for changes that still need to be made: 

1. I wanted to confirm that the below photo info edits are being made on all 3 versions of the subway intro page:
--The photo has to follow all the standards listed here
--header change to "Examples of good ID photos"
--New change:  the first bullet under the photo standards should be updated to: 
Show a full front view of your face and neck (with no hat, head covering, or headphones covering or casting shadows on your hairline or face), **and**


2. Sign in to Vets.gov and verify your identity (lowercase: "in" and "verify your identity")

3. Add the word "proofing" here: Once you've gone through the identity-proofing process, you won't need to do it again. 

4. Please complete the Veteran ID Card form to apply for a card.

**Pull request merged**